src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,13 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px #{$catalog-padding-vertical};
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  width: catalogwidth (
```

Comment:
It does not work because you have a leftover space after function calling :)
![image](https://user-images.githubusercontent.com/25587299/209981659-175227c6-84f9-470b-8039-c728eadb31e4.png)